Tests/Specs/versioned-api/tests/transaction-handling.json

```diff
@@ -6,7 +6,8 @@
       "minServerVersion": "4.9",
       "topologies": [
         "replicaset",
-        "sharded-replicaset"
```

Comment:
curiously, this file did not have its schema version bumped, even though I think support for load-balanced topology probably would have warranted that? in any case, this file was just synced directly.

Reply:
Yeah this should be bumped to 1.3 for load balancer I believe. Probably worth filing a PR/DRIVERS ticket for that.

Reply:
opened mongodb/specifications#1023